code/game/antagonist/outsider/hacker.dm

```diff
@@ -0,0 +1,45 @@
+var/datum/antagonist/hacker/hacker
+
+/datum/antagonist/hacker
+	id = MODE_CLANDESTINE
+	bantype = "Hacker (Clandestine)"
+	role_type = BE_HACKER
+	role_text = "Hacker"
+	role_text_plural = "Elite Hacking Group"
+	welcome_text = "<b>Be prepared to help the spy any way you can!<b>"
```

Comment:
close with `</b>` not `<b>`

Reply:
Ahh. Wish BYOND compiler had of picked that up.